src/main/java/winstone/AbstractSecuredConnectorFactory.java

```diff
@@ -197,14 +199,20 @@ protected SslContextFactory getSSLContext( Map args) {
             }
 
             SslContextFactory ssl = new SslContextFactory();
-
             ssl.setKeyStore(keystore);
             ssl.setKeyStorePassword(keystorePassword);
             ssl.setKeyManagerPassword(privateKeyPassword);
             ssl.setKeyManagerFactoryAlgorithm(Option.HTTPS_KEY_MANAGER_TYPE.get(args));
             ssl.setCertAlias(Option.HTTPS_CERTIFICATE_ALIAS.get(args));
             ssl.setExcludeProtocols("SSLv3", "SSLv2", "SSLv2Hello");
-
+            String excludeCiphers = Option.HTTPS_EXCLUDE_CIPHER_SUITES.get( args );
+            if( StringUtil.isNotBlank(excludeCiphers)) {
+                String[] cipherSuites = excludeCiphers.split(",");
```

Comment:
```suggestion
                String[] cipherSuites = excludeCiphers.split("\\s*,\\s*");
```

Reply:
Note that it will not be handled properly if the user-specified regexp includes `,` due to whatever reason

Reply:
ok (well not sure to have cipher name including `,` :) but it's a very simple change

Reply:
Correct me if I am wrong, but if I pass: `RSA_X,RSA_Y` (2x), it will pass as expected (before and after). But if I pass `RSA_X,Y_OR_Z` (1x), the behavior is exactly the same, even for `RSA_X, Y_OR_Z` (1x).

The only thing that is changed is in case I pass `RSA_X, RSA_Y` (2x) with the space after the comma, before the patch, the second regex contains the comma and after the correction it will not. So the correction is about user-specified regexp separated by `,` with spaces, not directly in the regexp right?

Please propose samples that show the faulty behavior if my hypothesis is not correct.

Reply:
@Wadeck you're correct I'm not sure how to handle `,` but I'm not sure if we have to manage this as I cannot really find any cipher with `,` in his name... See https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml#tls-parameters-4
So I would simply use the first version with `  String[] cipherSuites = excludeCiphers.split(","); `
and voila no real need here to make this more complicated...
?